backend/database.js

```diff
@@ -18,12 +18,13 @@ const production = process.env.NODE_ENV === 'production'
 const persistence = process.env.GI_PERSIST || (production ? 'fs' : undefined)
 
 // Default database options. Other values may be used e.g. in tests.
+const dbRootPath = process.env.DB_ROOT_PATH || ''
 const options = {
   fs: {
-    dirname: './data'
+    dirname: `.${dbRootPath}/data`
```

Comment:
I'm not a fan of the `.${dbRootPath}` thing and would replace it with `${dbRootPath}`.